Fix rx-netty/examples build.

tbak opened the pull request.

Commit:
Fix rx-netty/examples build.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #67](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/67/) SUCCESS
This pull request looks good

Commit:
Move examples from rx-netty/src/examples to own top level project.
Given the need to write some unit tests for the examples, this will
make the build setup more explicite (versus now being custom
configuration of rx-netty project).

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #68](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/68/) SUCCESS
This pull request looks good

NiteshKant commented:
Does this also publish examples as an artifact to maven?
I would prefer the entire examples revamp (issue #127 ) as a single PR over the incremental changes. Do you want your incremental changes to be merged?

NiteshKant commented:
Discussed offline with @tbak. He will send a different PR with the comprehensive changes for issue #127. Closing this PR.
